Bumping EUI to 39.0.0

breehall opened the pull request:
## Summary
eui@38.0.1 ⏩ ~eui@38.2.0~ eui@39.0.0

## [`39.0.0`](https://github.com/elastic/eui/tree/v39.0.0)

- Added `maxWidth` prop to `EuiTour`, made `subtitle` optional, and fixed heading levels and footer background ([#5225](https://github.com/elastic/eui/pull/5225))
- Updated `tint`, `shade`, `saturate`, `desaturate`, and `makeHighContrastColor` utility functions to maintain color format supplied ([#5230](https://github.com/elastic/eui/pull/5230))
- Converted generated icon files to Typescript, eliminating the last `.js` files in our source files ([#5212](https://github.com/elastic/eui/pull/5212))

**Bug fixes**

- Fixed tick and level alignment in `Eui[Dual]Range` ([#5181](https://github.com/elastic/eui/pull/5181))
- Fixed multiple accessibility issues in `EuiBasicTable` and `EuiInMemoryTable` ([#5237](https://github.com/elastic/eui/pull/5237), [#5241](https://github.com/elastic/eui/pull/5241))
- Fixed missing i18n token in `EuiBasicTable`'s no items message ([#5242](https://github.com/elastic/eui/pull/5242))

**Breaking changes**

- Removed `boolean` from `EuiTour`'s `minWidth` type ([#5225](https://github.com/elastic/eui/pull/5225))

## [`38.2.0`](https://github.com/elastic/eui/tree/master)

- Updated `EuiRangeLevel` `color` property to accept CSS color values ([#5171](https://github.com/elastic/eui/pull/5171))

**Bug fixes**

- Fixed logo icons with static SVG IDs causing accessibility errors when multiples of the same logo were present ([#5204](https://github.com/elastic/eui/pull/5204))
- Fixed several `EuiDataGrid` console errors that occur on column drag/drop reorder ([#5209](https://github.com/elastic/eui/pull/5209))

**Reverts**

- Reverted `EuiScreenReaderOnly` left positioning change due to Selenium issues ([#5215](https://github.com/elastic/eui/pull/5215))

## [`38.1.0`](https://github.com/elastic/eui/tree/v38.1.0)

- Fixed the `title` prop `EuiButtonGroup` to automatically display the `label` provided ([#5199](https://github.com/elastic/eui/pull/5199))
- Updated `barSeriesStyle.displayValue` of the elastic-charts `Theme` for better default styles ([#4845](https://github.com/elastic/eui/pull/4845))
- Added a configuration parameter to the `EuiMarkdownEditor` plugin functions to exclude custom plugins ([#5147](https://github.com/elastic/eui/pull/5147))
- Added `auto` as value for `defaultHeight` in prop `rowHeightsOptions` in `EuiDataGrid` that allows to content auto-fit to row ([#4958](https://github.com/elastic/eui/pull/4958))
- Updated `titleProps` and `descriptionProps` on `EuiDescriptionList` to extend `CommonProps` ([#5166](https://github.com/elastic/eui/pull/5166))
- Added the ability to return `visibleOptions` from `EuiSelectable` by using `onSearch` ([#5178](https://github.com/elastic/eui/pull/5178))

**Bug fixes**

- Fixed `EuiDataGrid` focus ring to be contained in the cell ([#5194](https://github.com/elastic/eui/pull/5194))
- Fixed `EuiDataGrid` cells when focused getting a higher `z-index` which was causing long content to overlap surrounding cells ([#5194](https://github.com/elastic/eui/pull/5194))
- Replaced the `EuiMarkdownEditor` help syntax modal with a popover when no custom plugins are available ([#5147](https://github.com/elastic/eui/pull/5147))
- Fixed multiple components unnecessarily rerendering generated IDs on every update ([#5195](https://github.com/elastic/eui/pull/5195), [#5196](https://github.com/elastic/eui/pull/5196), [#5197](https://github.com/elastic/eui/pull/5197), [#5200](https://github.com/elastic/eui/pull/#5200), [#5201](https://github.com/elastic/eui/pull/#5201))

**Theme: Amsterdam**

- Fixed `border-radius` and increased `font-weight` for `EuiButtonGroup` ([#4993](https://github.com/elastic/eui/pull/4993))
- Increased contrast of text `color` for `text` colored `EuiButton` and `EuiButtonIcon` ([#5177](https://github.com/elastic/eui/pull/5177))

breehall committed:
Upgraded the version of EUI to 38.2.0 from 38.0.1

breehall committed:
Updated the i18n mappings required for EUI v.38.2.0

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157127/)
* Commit: 69ee19c8d21856b16467effddfb9956dd3e41d16
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [Quick Commit Checks](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157127/execution/node/421/log)

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157127/","number":"157127","commit":"69ee19c8d21856b16467effddfb9956dd3e41d16"}]}
PIPELINE-->

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157672/)
* Commit: bfd7d5f753c00460218cfad7e552c0bf645419f3
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [Quick Commit Checks](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157672/execution/node/421/log)

### History
* :broken_heart: [Build #157127](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157127/) failed 69ee19c8d21856b16467effddfb9956dd3e41d16

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157127/","number":"157127","commit":"69ee19c8d21856b16467effddfb9956dd3e41d16"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157672/","number":"157672","commit":"bfd7d5f753c00460218cfad7e552c0bf645419f3"}]}
PIPELINE-->

breehall committed:
Update i18n snapshots and resolve linting error

breehall committed:
Removed html_id_generator mocks.

Current mock was failing due to missing useGeneratedHtmlId export. This is safe to remove because EUI contains a .testenv that contains an mock for html_id_generator. More info at https://github.com/elastic/eui/blob/master/src/services/accessibility/html_id_generator.testenv.ts

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157683/)
* Commit: 8a29a960692dfdb63d8cf26db8d6f4096602f4ff
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [Execute xpack-APMCypress](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157683/execution/node/537/log)
* [Execute xpack-APMCypress](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157683/execution/node/697/log)

### History
* :broken_heart: [Build #157672](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157672/) failed bfd7d5f753c00460218cfad7e552c0bf645419f3
* :broken_heart: [Build #157127](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157127/) failed 69ee19c8d21856b16467effddfb9956dd3e41d16

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157127/","number":"157127","commit":"69ee19c8d21856b16467effddfb9956dd3e41d16"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157672/","number":"157672","commit":"bfd7d5f753c00460218cfad7e552c0bf645419f3"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157683/","number":"157683","commit":"8a29a960692dfdb63d8cf26db8d6f4096602f4ff"}]}
PIPELINE-->

breehall committed:
Resolve linting error in i18n mapping file

breehall committed:
Removed html_id_generator mocks.

Current mock was failing due to missing useGeneratedHtmlId export. This is safe to remove because EUI contains a .testenv that contains a mock for html_id_generator. More info at https://github.com/elastic/eui/blob/master/src/services/accessibility/html_id_generator.testenv.ts

breehall committed:
Update plugin snapshots

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157780/)
* Commit: ca1991e0774d16de435f24e0ed87da420b645e0d
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [Execute xpack-APMCypress](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157780/execution/node/533/log)
* [Execute xpack-accessibility](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157780/execution/node/547/log)

### History
* :broken_heart: [Build #157683](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157683/) failed 8a29a960692dfdb63d8cf26db8d6f4096602f4ff
* :broken_heart: [Build #157672](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157672/) failed bfd7d5f753c00460218cfad7e552c0bf645419f3
* :broken_heart: [Build #157127](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157127/) failed 69ee19c8d21856b16467effddfb9956dd3e41d16

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157127/","number":"157127","commit":"69ee19c8d21856b16467effddfb9956dd3e41d16"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157672/","number":"157672","commit":"bfd7d5f753c00460218cfad7e552c0bf645419f3"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157683/","number":"157683","commit":"8a29a960692dfdb63d8cf26db8d6f4096602f4ff"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157780/","number":"157780","commit":"ca1991e0774d16de435f24e0ed87da420b645e0d"}]}
PIPELINE-->

breehall committed:
Merging in the latest code from the master branch and resolve merge conflict in availability_reporting.test.tsx

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158128/)
* Commit: eea820452cb7ef8a715502ad8a0a5c5a5dfc66ed
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [Check Licenses](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158128/execution/node/432/log)

### History
* :broken_heart: [Build #157780](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157780/) failed ca1991e0774d16de435f24e0ed87da420b645e0d
* :broken_heart: [Build #157683](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157683/) failed 8a29a960692dfdb63d8cf26db8d6f4096602f4ff
* :broken_heart: [Build #157672](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157672/) failed bfd7d5f753c00460218cfad7e552c0bf645419f3
* :broken_heart: [Build #157127](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157127/) failed 69ee19c8d21856b16467effddfb9956dd3e41d16

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157127/","number":"157127","commit":"69ee19c8d21856b16467effddfb9956dd3e41d16"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157672/","number":"157672","commit":"bfd7d5f753c00460218cfad7e552c0bf645419f3"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157683/","number":"157683","commit":"8a29a960692dfdb63d8cf26db8d6f4096602f4ff"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157780/","number":"157780","commit":"ca1991e0774d16de435f24e0ed87da420b645e0d"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158128/","number":"158128","commit":"eea820452cb7ef8a715502ad8a0a5c5a5dfc66ed"}]}
PIPELINE-->

breehall committed:
Resolve merge conflict in license_checker config.ts file

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158152/)
* Commit: 308c7ca39347b52628a3bb65d49cf75292ebe806
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [Execute xpack-UptimePlaywright](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158152/execution/node/538/log)
* [Execute xpack-UptimePlaywright](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158152/execution/node/687/log)

### History
* :broken_heart: [Build #158128](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158128/) failed eea820452cb7ef8a715502ad8a0a5c5a5dfc66ed
* :broken_heart: [Build #157780](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157780/) failed ca1991e0774d16de435f24e0ed87da420b645e0d
* :broken_heart: [Build #157683](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157683/) failed 8a29a960692dfdb63d8cf26db8d6f4096602f4ff
* :broken_heart: [Build #157672](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157672/) failed bfd7d5f753c00460218cfad7e552c0bf645419f3
* :broken_heart: [Build #157127](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157127/) failed 69ee19c8d21856b16467effddfb9956dd3e41d16

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157127/","number":"157127","commit":"69ee19c8d21856b16467effddfb9956dd3e41d16"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157672/","number":"157672","commit":"bfd7d5f753c00460218cfad7e552c0bf645419f3"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157683/","number":"157683","commit":"8a29a960692dfdb63d8cf26db8d6f4096602f4ff"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157780/","number":"157780","commit":"ca1991e0774d16de435f24e0ed87da420b645e0d"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158128/","number":"158128","commit":"eea820452cb7ef8a715502ad8a0a5c5a5dfc66ed"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158152/","number":"158152","commit":"308c7ca39347b52628a3bb65d49cf75292ebe806"}]}
PIPELINE-->

breehall committed:
Merge branch 'master' of https://github.com/elastic/kibana into eui-upgrade-38.2.0
Merging in the latest code from the Master branch

breehall committed:
Upgrade EUI to version 39.0.0 from the original target (38.2.0) to handle an issue found with a functional test during the original upgrade

breehall committed:
Updated the i18n mapping for EUI v.39.0.0

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158486/)
* Commit: 432b23f5e9efe4af66738dfd63dc1bdbf5a01f6b
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-113633/432b23f5e9efe4af66738dfd63dc1bdbf5a01f6b)
* This comment will update when the build is complete

### Failed CI Steps
* [Jest Unit Tests](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158486/execution/node/190/log)

### History
* :broken_heart: [Build #158152](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158152/) failed 308c7ca39347b52628a3bb65d49cf75292ebe806
* :broken_heart: [Build #158128](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158128/) failed eea820452cb7ef8a715502ad8a0a5c5a5dfc66ed
* :broken_heart: [Build #157780](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157780/) failed ca1991e0774d16de435f24e0ed87da420b645e0d
* :broken_heart: [Build #157683](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157683/) failed 8a29a960692dfdb63d8cf26db8d6f4096602f4ff
* :broken_heart: [Build #157672](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157672/) failed bfd7d5f753c00460218cfad7e552c0bf645419f3

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157672/","number":"157672","commit":"bfd7d5f753c00460218cfad7e552c0bf645419f3"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157683/","number":"157683","commit":"8a29a960692dfdb63d8cf26db8d6f4096602f4ff"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157780/","number":"157780","commit":"ca1991e0774d16de435f24e0ed87da420b645e0d"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158128/","number":"158128","commit":"eea820452cb7ef8a715502ad8a0a5c5a5dfc66ed"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158152/","number":"158152","commit":"308c7ca39347b52628a3bb65d49cf75292ebe806"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158486/","number":"158486","commit":"432b23f5e9efe4af66738dfd63dc1bdbf5a01f6b"}]}
PIPELINE-->

breehall committed:
Update various snapshots to account for the an i18n translation token addition in EUI v. 39.0.0

breehall committed:
Updated test cases marked as obsolete by CI

breehall committed:
Merge branch 'master' of https://github.com/elastic/kibana into eui-upgrade-38.2.0
Merging in the latest code from the Master branch

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/159003/)
* Commit: f6833eb27424a7af0b963ff679094803234679cf
* Storybooks not built yet
* This comment will update when the build is complete

### Failed CI Steps
* [Execute xpack-kibana-ciGroupDocker](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/159003/execution/node/731/log)
* [Execute xpack-kibana-ciGroupDocker](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/159003/execution/node/747/log)

### History
* :broken_heart: [Build #158627](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158627/) failed fb88b043e1ddbaddead965e155376411ae8d3c28
* :broken_heart: [Build #158486](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158486/) failed 432b23f5e9efe4af66738dfd63dc1bdbf5a01f6b
* :broken_heart: [Build #158152](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158152/) failed 308c7ca39347b52628a3bb65d49cf75292ebe806
* :broken_heart: [Build #158128](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158128/) failed eea820452cb7ef8a715502ad8a0a5c5a5dfc66ed
* :broken_heart: [Build #157780](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157780/) failed ca1991e0774d16de435f24e0ed87da420b645e0d

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/157780/","number":"157780","commit":"ca1991e0774d16de435f24e0ed87da420b645e0d"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158128/","number":"158128","commit":"eea820452cb7ef8a715502ad8a0a5c5a5dfc66ed"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158152/","number":"158152","commit":"308c7ca39347b52628a3bb65d49cf75292ebe806"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158486/","number":"158486","commit":"432b23f5e9efe4af66738dfd63dc1bdbf5a01f6b"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158627/","number":"158627","commit":"fb88b043e1ddbaddead965e155376411ae8d3c28"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/159003/","number":"159003","commit":"f6833eb27424a7af0b963ff679094803234679cf"}]}
PIPELINE-->

kibanamachine committed:
Merge branch 'master' into eui-upgrade-38.2.0

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/159133/)
* Commit: 6a940f50aa76c36d90202433e06b60fda921c27b
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-113633/6a940f50aa76c36d90202433e06b60fda921c27b)
* [Documentation Changes](https://kibana_113633.docs-preview.app.elstc.co/diff)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/e62b7525168d2d65182ff85b71bf00d85a8e2a35) | [after](https://github.com/elastic/kibana/commit/6a940f50aa76c36d90202433e06b60fda921c27b) | diff |
| --- | --- | --- | --- |
| `apm` | 2.7MB | 2.7MB | -1.0B |
| `canvas` | 1.0MB | 1.0MB | -1.0B |
| `cloud` | 12.3KB | 12.3KB | -1.0B |
| `enterpriseSearch` | 1.4MB | 1.4MB | -1.0B |
| `fileUpload` | 673.4KB | 673.4KB | -1.0B |
| `graph` | 403.3KB | 403.3KB | -1.0B |
| `infra` | 938.4KB | 938.4KB | -1.0B |
| `ingestPipelines` | 453.1KB | 453.1KB | -1.0B |
| `kibanaReact` | 203.5KB | 203.5KB | -2.0B |
| `lens` | 1.0MB | 1.0MB | -1.0B |
| `maps` | 2.7MB | 2.7MB | -1.0B |
| `mapsEms` | 174.0KB | 174.0KB | -1.0B |
| `ml` | 3.6MB | 3.6MB | -1.0B |
| `reporting` | 43.9KB | 43.9KB | -1.0B |
| `security` | 514.6KB | 514.6KB | -1.0B |
| `securitySolution` | 4.3MB | 4.3MB | -1.0B |
| `uiActionsEnhanced` | 128.8KB | 128.8KB | -1.0B |
| `uptime` | 568.6KB | 568.6KB | -1.0B |
| `visTypeTimeseries` | 485.9KB | 485.9KB | -1.0B |
| `visTypeVega` | 1.9MB | 1.9MB | -2.0B |
| `visTypeVislib` | 373.7KB | 373.7KB | -1.0B |
| total |  |  | -23.0B |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/e62b7525168d2d65182ff85b71bf00d85a8e2a35) | [after](https://github.com/elastic/kibana/commit/6a940f50aa76c36d90202433e06b60fda921c27b) | diff |
| --- | --- | --- | --- |
| `core` | 300.3KB | 300.8KB | +468.0B |
| `fieldFormats` | 49.0KB | 49.0KB | -1.0B |
| `fleet` | 103.4KB | 103.4KB | -1.0B |
| `kbnUiSharedDeps-css` | 596.7KB | 597.0KB | +321.0B |
| `kbnUiSharedDeps-npmDll` | 5.1MB | 5.1MB | +18.3KB |
| `kbnUiSharedDeps-srcJs` | 3.8MB | 3.8MB | -1.0B |
| `kibanaUtils` | 85.8KB | 85.7KB | -1.0B |
| `share` | 54.0KB | 54.0KB | -1.0B |
| total |  |  | +19.1KB |

### History
* :broken_heart: [Build #159003](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/159003/) failed f6833eb27424a7af0b963ff679094803234679cf
* :broken_heart: [Build #158627](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158627/) failed fb88b043e1ddbaddead965e155376411ae8d3c28
* :broken_heart: [Build #158486](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158486/) failed 432b23f5e9efe4af66738dfd63dc1bdbf5a01f6b
* :broken_heart: [Build #158152](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158152/) failed 308c7ca39347b52628a3bb65d49cf75292ebe806
* :broken_heart: [Build #158128](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158128/) failed eea820452cb7ef8a715502ad8a0a5c5a5dfc66ed

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158128/","number":"158128","commit":"eea820452cb7ef8a715502ad8a0a5c5a5dfc66ed"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158152/","number":"158152","commit":"308c7ca39347b52628a3bb65d49cf75292ebe806"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158486/","number":"158486","commit":"432b23f5e9efe4af66738dfd63dc1bdbf5a01f6b"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158627/","number":"158627","commit":"fb88b043e1ddbaddead965e155376411ae8d3c28"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/159003/","number":"159003","commit":"f6833eb27424a7af0b963ff679094803234679cf"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/159133/","number":"159133","commit":"6a940f50aa76c36d90202433e06b60fda921c27b"}]}
PIPELINE-->

elasticmachine commented:
Pinging @elastic/eui-design (EUI)

elasticmachine commented:
Pinging @elastic/apm-ui (Team:apm)

elasticmachine commented:
Pinging @elastic/uptime (Team:uptime)

mattkime reviewed (approved):
App Services changes lgtm

breehall committed:
Update src/dev/license_checker/config.ts

Removing TODO comments from src/dev/license_checker/config.ts as they are no longer needed.

Co-authored-by: Constance <constancecchen@users.noreply.github.com>

kibanamachine commented:
## :green_heart: Build Succeeded
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/159244/)
* Commit: 379acdda3f5b25fce54718a7703a79287c417268
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-113633/379acdda3f5b25fce54718a7703a79287c417268)
* [Documentation Changes](https://kibana_113633.docs-preview.app.elstc.co/diff)

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/e62b7525168d2d65182ff85b71bf00d85a8e2a35) | [after](https://github.com/elastic/kibana/commit/379acdda3f5b25fce54718a7703a79287c417268) | diff |
| --- | --- | --- | --- |
| `apm` | 2.7MB | 2.7MB | -1.0B |
| `canvas` | 1.0MB | 1.0MB | -1.0B |
| `cloud` | 12.3KB | 12.3KB | -1.0B |
| `enterpriseSearch` | 1.4MB | 1.4MB | -1.0B |
| `fileUpload` | 673.4KB | 673.4KB | -1.0B |
| `graph` | 403.3KB | 403.3KB | -1.0B |
| `infra` | 938.4KB | 938.4KB | -1.0B |
| `ingestPipelines` | 453.1KB | 453.1KB | -1.0B |
| `kibanaReact` | 203.5KB | 203.5KB | -2.0B |
| `lens` | 1.0MB | 1.0MB | -1.0B |
| `maps` | 2.7MB | 2.7MB | -1.0B |
| `mapsEms` | 174.0KB | 174.0KB | -1.0B |
| `ml` | 3.6MB | 3.6MB | -1.0B |
| `reporting` | 43.9KB | 43.9KB | -1.0B |
| `security` | 514.6KB | 514.6KB | -1.0B |
| `securitySolution` | 4.3MB | 4.3MB | -1.0B |
| `uiActionsEnhanced` | 128.8KB | 128.8KB | -1.0B |
| `uptime` | 568.6KB | 568.6KB | -1.0B |
| `visTypeTimeseries` | 485.9KB | 485.9KB | -1.0B |
| `visTypeVega` | 1.9MB | 1.9MB | -2.0B |
| `visTypeVislib` | 373.7KB | 373.7KB | -1.0B |
| total |  |  | -23.0B |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/e62b7525168d2d65182ff85b71bf00d85a8e2a35) | [after](https://github.com/elastic/kibana/commit/379acdda3f5b25fce54718a7703a79287c417268) | diff |
| --- | --- | --- | --- |
| `core` | 300.3KB | 300.8KB | +468.0B |
| `fieldFormats` | 49.0KB | 49.0KB | -1.0B |
| `fleet` | 103.4KB | 103.4KB | -1.0B |
| `kbnUiSharedDeps-css` | 596.7KB | 597.0KB | +321.0B |
| `kbnUiSharedDeps-npmDll` | 5.1MB | 5.1MB | +18.3KB |
| `kbnUiSharedDeps-srcJs` | 3.8MB | 3.8MB | -1.0B |
| `kibanaUtils` | 85.8KB | 85.7KB | -1.0B |
| `share` | 54.0KB | 54.0KB | -1.0B |
| total |  |  | +19.1KB |

### History
* :green_heart: [Build #159133](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/159133/) succeeded 6a940f50aa76c36d90202433e06b60fda921c27b
* :broken_heart: [Build #159003](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/159003/) failed f6833eb27424a7af0b963ff679094803234679cf
* :broken_heart: [Build #158627](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158627/) failed fb88b043e1ddbaddead965e155376411ae8d3c28
* :broken_heart: [Build #158486](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158486/) failed 432b23f5e9efe4af66738dfd63dc1bdbf5a01f6b
* :broken_heart: [Build #158152](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158152/) failed 308c7ca39347b52628a3bb65d49cf75292ebe806

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158152/","number":"158152","commit":"308c7ca39347b52628a3bb65d49cf75292ebe806"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158486/","number":"158486","commit":"432b23f5e9efe4af66738dfd63dc1bdbf5a01f6b"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/158627/","number":"158627","commit":"fb88b043e1ddbaddead965e155376411ae8d3c28"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/159003/","number":"159003","commit":"f6833eb27424a7af0b963ff679094803234679cf"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/159133/","number":"159133","commit":"6a940f50aa76c36d90202433e06b60fda921c27b"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/159244/","number":"159244","commit":"379acdda3f5b25fce54718a7703a79287c417268"}]}
PIPELINE-->

sabarasaba reviewed (approved):
stack management changes LGTM 🚀  didn't test locally

jbudz reviewed (approved):
license checker config LGTM

smith reviewed (approved):
APM changes look good

thompsongl commented:
@elasticmachine merge upstream

kibanamachine committed:
Merge branch 'master' into eui-upgrade-38.2.0

thompsongl commented:
@elasticmachine merge upstream

kibanamachine committed:
Merge branch 'master' into eui-upgrade-38.2.0

kibanamachine committed:
Merge branch 'master' into eui-upgrade-38.2.0

thompsongl commented:
@elasticmachine merge upstream

kibanamachine committed:
Merge branch 'master' into eui-upgrade-38.2.0

kibanamachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [continuous-integration/kibana-ci/pull-request](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/159796/)
* Commit: f2394b27290cb0ac222b4685d85a8a5f4538d0ec
* [Storybooks Preview](https://ci-artifacts.kibana.dev/storybooks/pr-113633/f2394b27290cb0ac222b4685d85a8a5f4538d0ec)
* This comment will update when the build is complete

### Failed CI Steps
* [Execute xpack-securitySolutionCypressChrome](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/159796/execution/node/529/log)
* [Execute xpack-securitySolutionCypressChrome](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/159796/execution/node/1078/log)

### History
* :broken_heart: [Build #159720](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/159720/) failed d13120aca966232af983b304f1a16f5f4aad3a09
* :broken_heart: [Build #159645](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/159645/) failed dd4ccbb515ebcff8f2f9031e816d4573993a478a
* :broken_heart: [Build #159598](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/159598/) failed 54726eaaef9bba61f80816c8ab1e14547d09ba9e
* :green_heart: [Build #159244](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/159244/) succeeded 379acdda3f5b25fce54718a7703a79287c417268
* :green_heart: [Build #159133](https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/159133/) succeeded 6a940f50aa76c36d90202433e06b60fda921c27b

To update your PR or re-run it, just comment with:
`@elasticmachine merge upstream`

<!--PIPELINE
{"builds":[{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/159133/","number":"159133","commit":"6a940f50aa76c36d90202433e06b60fda921c27b"},{"status":"SUCCESS","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/159244/","number":"159244","commit":"379acdda3f5b25fce54718a7703a79287c417268"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/159598/","number":"159598","commit":"54726eaaef9bba61f80816c8ab1e14547d09ba9e"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/159645/","number":"159645","commit":"dd4ccbb515ebcff8f2f9031e816d4573993a478a"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/159720/","number":"159720","commit":"d13120aca966232af983b304f1a16f5f4aad3a09"},{"status":"FAILURE","url":"https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/159796/","number":"159796","commit":"f2394b27290cb0ac222b4685d85a8a5f4538d0ec"}]}
PIPELINE-->

VladLasitsa commented:
@elasticmachine merge upstream

kibanamachine committed:
Merge branch 'master' into eui-upgrade-38.2.0

justinkambic reviewed (approved):
Uptime LGTM

breehall commented:
Merging this PR as the remaining reviews are snapshot related
